### PR TITLE
[chimera-pverify] Review

### PR DESCRIPTION
**Pull request by wschoenell**
I am going thru pverify and fixing all the bugs that I find and make it usable again. This version is a working version for start. A LOT should be improved, but it at least works now.

The main changes are:

* Added a `download` method to `ImageUtil` for downloading remote images. This is useful when you have the camera running on another computer than the one where you are running the pverify;

* Removed IRAF dependency;

* A lot of code organization, removed not-used imports, changed `print()`s by `log.debug()`s etc on (and only) pverify methods.

### CODE REVIEW

**Comment by ankanaan** [comment]
pverify is and always perfect :-)

File: setup.py [hunk 1]
```diff
@@ -70,7 +67,7 @@
     scripts=chimera_scripts,
 
     # installation happens in the specified order
-    install_requires=[
+    install_requires=[  "astropy",
```
Comment (phsilva) [inline setup.py]: duplicated? astropy is already included.

File: src/chimera/controllers/pointverify.py [hunk 2]
```diff
@@ -127,39 +125,30 @@ def pointVerify(self):
...
             #    self.checkedpointing = False
             #    raise CanSetScopeButNotThisField("Able to set scope, but unable to verify this field %s" %(currentImageCenter))
-        wcs = Image.fromFile(wcs_name)
-        ra_wcs_center = wcs["CRVAL1"] + \
-            (wcs["NAXIS1"] / 2.0 - wcs["CRPIX1"]) * wcs["CD1_1"]
-        dec_wcs_center = wcs["CRVAL2"] + \
-            (wcs["NAXIS2"] / 2.0 - wcs["CRPIX2"]) * wcs["CD2_2"]
-        currentWCS = Position.fromRaDec(Coord.fromD(ra_wcs_center),
-                                        Coord.fromD(dec_wcs_center))
+        wcs = fits.getheader(wcs_name)
```
Comment (phsilva) [inline src/chimera/controllers/pointverify.py]: maybe put this in a method on Image class? looks like something useful.

File: setup.py (hunk 1 above)
Comment (wschoenell) [inline setup.py]: I was betrayed by PyCharm!

File: src/chimera/controllers/pointverify.py (hunk 2 above)
Comment (wschoenell) [inline src/chimera/controllers/pointverify.py]: Done in 03fed072fd09f3c2665d95d7de5ad9a5a494ddb3

**Comment by wschoenell** [comment]
Added more changes:

* Change the filter to `filter` config directive file when exposing;
* Documented `pverify` interface. Changed `tolra` and `toldec` config parameters from `max_tolerance` which is equals to `sqrt(delta_ra**2 + delta_dec**2)`, `max_trials` is `max_tries` now for compatibility with the other interfaces;
* Fixed an issue of determining files base name when remote client is windows and local is linux and vice-versa;
* Added `worldAtCenter()` method to `Image` class;

**Comment by ankanaan** [comment]
On Mon, Aug 3, 2015 at 10:26 AM, William Schoenell <notifications@github.com
> wrote:

> Added more changes:
>
>    - Change the filter to filter config directive file when exposing;
>    - Documented pverify interface. Changed tolra and toldec config
>    parameters from max_tolerance which is equals to sqrt(delta_ra**2 +
>    delta_dec**2), max_trials is max_tries now for compatibility with the
>    other interfaces;
>
>
not sure I like this.  Telescopes often point better in one direction, in
which case it makes sense to have different tolerances - at least for
equatorial mounts.

**Comment by wschoenell** [comment]
@ankanaan,
Yes. I see your point. I just changed back to what it was on 5203d3b.

Thanks!

**Comment by wschoenell** [comment]
Merging this to afterwards move into a plugin

**Pull request merged**